sdk/core/core-http/review/core-http.api.md

```diff
@@ -100,7 +108,7 @@ export class BasicAuthenticationCredentials implements ServiceClientCredentials
 
 // @public
 export class BearerTokenAuthenticationPolicy extends BaseRequestPolicy {
-    constructor(nextPolicy: RequestPolicy, options: RequestPolicyOptions, credential: TokenCredential, scopes: string | string[], tokenCache: AccessTokenCache);
+    constructor(nextPolicy: RequestPolicy, options: RequestPolicyOptions, tokenCache: AccessTokenCache, tokenRefresher: AccessTokenRefresher);
```

Comment:
Would this be a breaking change?

Reply:
This might be fine because the libraries depend on `BearerTokenAuthenticationPolicy` indirectly through `bearerTokenAuthenticationPolicy` which is not being changed.